presto-docs/src/main/sphinx/functions/window.rst

```diff
@@ -17,6 +17,9 @@ clause to specify the window. A window has three components:
   to ``RANGE UNBOUNDED PRECEDING``, which is the same as
   ``RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW``. This frame contains all
   rows from the start of the partition up to the last peer of the current row.
+  In the absence of ``ORDER BY``, all rows are considered peers, so ``RANGE
```

Comment:
PSQL documentation has a bit simpler wording for that IMO:
https://www.postgresql.org/docs/9.3/sql-expressions.html#SYNTAX-WINDOW-FUNCTIONS

Reply:
I will keep current wording. I don't know whether we can borrow Postgres docs directly.
Perhaps we can (licensing), but this one sentence doesn't deserve spending more time on it IMO.
And, we can always improve later.